<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormDataWidget</class>
 <widget class="QWidget" name="FormDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>271</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Create Data</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="formProgressBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="estimatedTimeLabel">
      <property name="text">
       <string>Est time.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="startFormButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Start Form Input</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
